nix/default.nix

```diff
@@ -79,7 +78,6 @@ let
         pkgs.entr
       ] ++ docsPkgs;
     };
-  mls-test-cli = pkgs.mls-test-cli;
-  rusty-jwt-tools = pkgs.rusty-jwt-tools;
```

Comment:
Do we inherit the `rusty-jwt-tools` implicitly? Or, are they not required anymore?

Reply:
it's unused.

Reply:
![image](https://github.com/wireapp/wire-server/assets/40720523/36f41ea6-feeb-4f1b-b62f-f98aeca6c30c)